public/editor/scripts/editor/js/fc/bramble-ui-bridge.js

```diff
@@ -309,9 +301,11 @@ define(function(require) {
     // Preview Mode Toggle
     $("#preview-pane-nav-desktop").click(function() {
       activatePreviewMode("desktop");
+      analytics.event({ category : "EditorUI", action : "Preview Mode Toggle", label : "Desktop" });
```

Comment:
You slip into `EditorUI` here from `Editor UI`, which brings up a point I've wondered reading through this entire file: why don't you move these strings to some module and use properties instead, for example `AnalyticsCategory.EditorUI` or something?  Then you won't corrupt your data by making typos like this.